### DISCUSSION TITLE
No module named 'torch' - Torch is installed

### DISCUSSION THREAD

**Discussion by SoulSair** [hacksider/Deep-Live-Cam#234] [Q&A]
`SoulSair@DESKTOP-M6HKP15 MINGW64 ~/Documents/Deep-Live-Cam (main)
$ C:/Users/SoulSair/AppData/Local/Programs/Python/Python310/python.exe run.py
Traceback (most recent call last):
  File "C:\Users\SoulSair\Documents\Deep-Live-Cam\run.py", line 3, in <module>
    from modules import core
  File "C:\Users\SoulSair\Documents\Deep-Live-Cam\modules\core.py", line 14, in <module>
    import torch
ModuleNotFoundError: No module named 'torch'
`
![image](https://github.com/user-attachments/assets/b1529693-ba66-4c04-9de8-35787243383e)
![image](https://github.com/user-attachments/assets/e108a1d9-6046-48af-980b-7d2455415453)

**Comment by vlednerk** [comment]
pip install torch
req just fd up

**Comment by IMNotMax** [comment]
Had the same issue. 
Only able to work it out by creating a new venv for this project (and activating it) , and installing the requirement.txt
